frontend/src/components/custom-tools/output-for-doc-modal/OutputForDocModal.jsx

```diff
@@ -172,12 +175,14 @@ function OutputForDocModal({
   };
 
   const getAdapterInfo = () => {
-    axiosPrivate
-      .get(`/api/v1/unstract/${sessionDetails.orgId}/adapter/?adapter_type=LLM`)
-      .then((res) => {
-        const adapterList = res.data;
-        setAdapterData(getLLMModelNamesForProfiles(llmProfiles, adapterList));
-      });
+    let url = `/api/v1/unstract/${sessionDetails.orgId}/adapter/?adapter_type=LLM`;
```

Comment:
@harini-venkataraman "/api/v1/unstract/${sessionDetails.orgId}" can be defined in a path variable and reuse in repeating places. 
REF:- https://github.com/Zipstack/unstract/blob/1fb2d4e849e8ec0134050fb43db01597b98f4f52/frontend/src/components/deployments/api-deployment/api-deployments-service.js#L9

Reply:
@vishnuszipstack This change looks like a FE refactor, which we can scope it in a separate PR and refactor for all files. 
CC : @tahierhussain